var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -71,15 +71,31 @@ class PyScipy(PythonPackage):
 
     def build_args(self, spec, prefix):
         args = []
+        # From NumPy 1.10.0 on it's possible to do a parallel build.
+        if self.version >= Version('1.10.0'):
```

Comment:
This isn't needed, as far as I know, Scipy has always supported a parallel build. We might want to keep the stuff below though, at least for older versions of scipy. Depends on when the bug was fixed.

Reply:
The bug was in `numpy.disutils` so one can't infer from the scipy version when it was fixed. Is there a way to check what numpy version was used in the concretization ?

Reply:
```python
if self.spec.satifies('^py-numpy@1.2.3:'):
```
might work.